### DISCUSSION TITLE
Is it possible to make a "mirror" input?

### DISCUSSION THREAD

**Discussion by benesva4** [react-hook-form/react-hook-form#2116] [Q&A] [answered]
Hello, is it possible to make two uncontrolled inputs that act like a controlled with the shared value and onChange? In a way that if you write something to one, the second will show up the value and vice versa, while sharing a single state? So when you submit, they are a single value? Thank you.

**Comment by bluebill1049** [comment, marked as answer]
you mean this: https://codesandbox.io/s/dawn-field-t12y1?file=/src/App.js

**Reply by benesva4** [reply]
Exactly, but these will get two different names and values in the state. I know it's kinda niche thing to do :-/

**Reply by benesva4** [reply]
But I guess that I could separate them to two object and filter one out in handleSubmit method. I guess it would do. Thank you very much.

**Reply by bluebill1049** [reply]
you can register one of them as well, the other one u just need to hold the `ref` and do e.target.value :) simple

**Reply by bluebill1049** [reply]
> But I guess that I could separate them to two object and filter one out in handleSubmit method. I guess it would do. Thank you very much.

works too.

**Reply by benesva4** [reply]
> 
> 
> you can register one of them as well, the other one u just need to hold the `ref` and do e.target.value :) simple

I am not sure, if I understand.

**Reply by bluebill1049** [reply]
> > you can register one of them as well, the other one u just need to hold the `ref` and do e.target.value :) simple
> 
> I am not sure, if I understand.

here you go: https://codesandbox.io/s/dawn-field-t12y1?file=/src/App.js

**Reply by benesva4** [reply]
Awesome. Thank you very much!

**Reply by oherik** [reply]
> here you go: https://codesandbox.io/s/dawn-field-t12y1?file=/src/App.js

Hey @bluebill1049, this is a very old thread but is that codesandbox link still correct? I don't really see how it relates to `ref` and how you describe its usage.
